selftests/functional/test_basic.py

```diff
@@ -50,6 +50,17 @@ def test(self):
         self.log.info(hello())
 '''
 
+INVALID_PYTHON_TEST = '''
+from avocado import Test
+
+class MyTest(Test):
+
+    non_existing_variable_causing_crash
+
+    def test(self):
```

Comment:
While on it, would you please improve this functional check by using `def test_my_name` and verifying the result contains this as the test name? That could save us future troubles...